mods.json

```diff
@@ -1,5 +1,13 @@
 {
     "mods" : {
+        "ccpreset" : {
+            "name" : "Preset Revival",
+            "description" : "This brings back the preset menu, which allows you to start the game at a specific point.",
+            "page" : [{ "name" : "GitHub", "url" : "https://github.com/CCDirectLink/CCPresetRevival" }],
+            "archive_link" : "https://github.com/CCDirectLink/CCPresetRevival/archive/master.zip",
+            "hash" : { "sha256" : "b140da004846a75c859d0f44de1fff56d917fe43c8955f4d2564c79aa5ce84a8" },
```

Comment:
Hash does not match since master has changed since PR was opened

Reply:
Fixed.